evaluation-tasks/coughvid.py

```diff
@@ -117,13 +151,32 @@ def name(self):
     def run(self):
         # Really coughvid? webm + ogg?
         audiofiles = list(
-            glob.glob(os.path.join(self.requires().workdir, "public_dataset/*.webm"))
-            + glob.glob(os.path.join(self.requires().workdir, "public_dataset/*.ogg"))
+            glob.glob(os.path.join(self.requires()[0].workdir, "public_dataset/*.webm"))
+            + glob.glob(
+                os.path.join(self.requires()[0].workdir, "public_dataset/*.ogg")
+            )
+        )
+
+        labeldf = pd.read_csv(
+            os.path.join(self.requires()[1].workdir, "metadata.csv"),
+            header=None,
+            names=["filename", "label"],
         )
+        filename_with_labels = list(labeldf["filename"].to_numpy())
+        assert len(filename_with_labels) == len(set(filename_with_labels))
+        filename_with_labels = set(filename_with_labels)
+
```

Comment:
I'm wondering if this should be split into two tasks:
1) Find filenames with labels and symlink them.
2) Subsample to number of audio files, but based upon only filenames with labels

Reply:
I'm favor of this. Would be nice to generally try to have smaller, single objective tasks

Reply:
https://github.com/neuralaudio/hear2021-eval-kit/issues/24